pkg/compiler/syscall.go

```diff
@@ -35,25 +29,12 @@ var syscalls = map[string]map[string]string{
 		"Deserialize":  "Neo.Runtime.Deserialize",
 	},
 	"blockchain": {
-		"GetAccount":              "Neo.Blockchain.GetAccount",
 		"GetBlock":                "System.Blockchain.GetBlock",
-		"GetContract":             "Neo.Blockchain.GetContract",
-		"GetHeader":               "Neo.Blockchain.GetHeader",
-		"GetHeight":               "Neo.Blockchain.GetHeight",
+		"GetContract":             "System.Blockchain.GetContract",
+		"GetHeight":               "System.Blockchain.GetHeight",
```

Comment:
Well, these are not about removal, so not a part of 6e373c82cee1dc789457845790c4efee7d0ed94b.

Reply:
Fixed.